Framework/MDAlgorithms/src/ConvertToDiffractionMDWorkspace2.cpp

```diff
@@ -100,12 +104,13 @@ void ConvertToDiffractionMDWorkspace2::init() {
       "order to merge them later\n");
   setPropertyGroup("MinRecursionDepth", getBoxSettingsGroupName());
 
-  std::vector<double> extents{-50, +50};
+  std::vector<double> extents;
   declareProperty(
```

Comment:
I was on the fence. On the one hand this isn't a huge change to the inner workings of the algorithm, but it does effect the algorithm's default parameters and therefore default behaviour. With fresh eyes this morning I'd be more inclined to move to v3.